src/components/Integrations/BigQuery/FreeAccount/index.ts

```diff
@@ -0,0 +1,2 @@
+import FreeAccount from './FreeAccount';
```

Comment:
Why do you import and export in this file? Is it not enough with the export in `FreeAccount.tsx`?

Reply:
Comment accepted and applied